app/components/Header/DropdownContents/EventsDropdown/index.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import Icon from 'app/components/Icon';
+import { Link } from 'react-router-dom';
+import styles from './EventsDropdown.css';
+import shared from 'app/components/Header/Header.css';
+
+const EventsDropdown = () => (
+  <div className={styles.eventsDropdownEl}>
+    <div className={shared.dropdownSection} data-first-dropdown-section>
+      <Link className={styles.dropdownLink} to="/events">
+        <Icon className={styles.icon} name="list" prefix="ion-md-" size={25} />
+        <span>Liste</span>
+      </Link>
+      <Link className={styles.dropdownLink} to="/events/calendar">
+        <Icon
+          className={styles.icon}
+          name="calendar"
+          prefix="ion-md-"
+          size={25}
+        />
+        <span>Kalender</span>
+      </Link>
+    </div>
+    <div className={shared.dropdownSection}>
+      <Link className={shared.bottomLink} to="/users/me">
+        Dine arrangementer
+      </Link>
```

Comment:
Why is there a span in the one space but not the other?

Reply:
I'm not sure what you mean here, and what is this "other"? 🤔 Do you simply not like the use of `<span>`?